wrench/src/parse_function.rs

```diff
@@ -0,0 +1,90 @@
+use std::str::CharIndices;
+// A crapy parser for parsing strings like "translate(1, 3)"
+
+pub fn parse_function(s: &str) -> (&str, Vec<&str>) {
+    // XXX: This it not particular easy to read. Sorry.
+    struct Parser<'a> {
+        itr: CharIndices<'a>,
+        start: usize,
+        o: Option<<CharIndices<'a> as Iterator>::Item>,
+    }
+    impl<'a> Parser<'a> {
+        fn skip_whitespace(&mut self) {
+            while let Some(k) = self.o {
```

Comment:
I'll still look into this but it would be nice to get merge this functionality now as I have reftests that depend on it and I've already spent more time on this function than I want to.

I didn't know about peekable iterators so I'll give them a look.